nav2-usability/scenario-code/dynamic-linking/dynamic_linking_vrouter.dart

```diff
@@ -0,0 +1,132 @@
+// Copyright 2021, the Flutter project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// Dynamic linking example
+/// Done using VRouter
+
+import 'dart:math';
+
+import 'package:flutter/material.dart';
+import 'package:vrouter/vrouter.dart';
+
+void main() {
+  runApp(WishlistApp());
```

Comment:
I get a `Bad state: No element` exception when I run this sample (using the `dev` branch of `vrouter`.)

Reply:
Solved in https://github.com/flutter/uxr/pull/28/commits/02fb450fd3d4774214dae457ae76e472b19fba85
The GlobalKey was not given to VRouter.